README.md

```diff
@@ -1,12 +1,12 @@
 # Evaluations
 
-This repository contains all the Evidenced Arguments filed as part of the Retention and Promotion of members of the Polkadot Technical Fellowship. These Arguments have been submitted for the discussion and review of individual members' contributions and achievements, as well as for the Fellowship's on-chain bodies to signal approval or disapproval of.
+This repository contains all the Arguments filed as part of the Retention and Promotion of members of the Polkadot Technical Fellowship. These Arguments have been submitted for the discussion and review of individual members' contributions and achievements, as well as for the Fellowship's on-chain bodies to signal approval or disapproval of.
 
 More information about Membership management can be found on the [Polkadot Technical Fellowship Dashboard](https://polkadot-fellows.xyz/#/membership).
 
 ## History
 
-"Evidenced Arguments" used to be called simply "evidence" and you may sometimes see this defunct term used. It was deprecated in favour of Evidenced Arguments (or simply Arguments) since people misconstrued their responsibilities regarding this information and tended to simply dump a dispassionate list of code changes. THIS IS UNACCEPTABLE.
+Arguments used to be called simply "Evidence" and you may sometimes see this defunct term used. It was deprecated in favour of "Arguments" since people misconstrued their responsibilities regarding this information and tended to simply submit a list of code changes, which is unacceptable.
```

Comment:
```suggestion
Arguments used to be called simply "Evidence" and you may sometimes see this defunct term used. It was deprecated in favour of "Arguments", since people misconstrued their responsibilities regarding this information and tended to simply submit a list of code changes.
```